neuro_extras/main.py

```diff
@@ -1032,11 +1030,11 @@ async def _run_copy_container(src_cluster: str, src_path: str, dst_path: str) ->
         "cpu-small",
         "--pass-config",
         "-v",
-        "storage:://storage",
+        f"{dst_path}://storage",
```

Comment:
it's an old thing, but why do we have `//storage` not `/storage`?

Reply:
IDK, to be honset. Here I just followed an old proverb: "do not touch the things, that are working" :D but It would be great to know, why do we put double `//` instead of single `/`.

@anayden, maybe you know that?

Reply:
ideally, dst_path should be named sth like dst_uri, but its' really minor, so lets merge!